src/components/NewTodo/NewTodo.tsx

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
```

Comment:
Consider removing the 'eslint-disable' comment and add an 'aria-label' attribute to the button to improve accessibility.